### PR TITLE
Refreshed all hair styles + new hair style

### PR DESCRIPTION
**Pull request by RaionArt**
Mostly fixing minor mistakes and adding one new hair style

### CODE REVIEW

File: game/outfits.rpy [hunk 1]
```diff
@@ -1111,6 +1111,12 @@ init -1 python in jn_outfits:
         unlocked=False,
         is_jn_wearable=True
     ))
+    __register_wearable(JNHairstyle(
+        reference_name="jn_hair_twintails_long",
+        display_name="Twintails long",
```
Comment (Blizzardsev) [inline game/outfits.rpy:1116]: I think for this one, we might want to call it "Long hair with twintails", just to make it clear from the name that it isn't Natsuki's twintails that are long specifically!

**Review by Blizzardsev — CHANGES_REQUESTED** [review]
No issues at all art-wise!

Only a small naming thing I can think of for the new hairstyle, otherwise I think it's fine!

**Comment by Blizzardsev** [comment]
@Nemu-sus Would love your input on this when you can!

File: game/outfits.rpy (hunk 1 above)
Comment (RaionArt) [reply game/outfits.rpy:1116]: I tried to imitate the name style like in the other case of the long hair version, but I can adjust it if you prefer it that way.

File: game/outfits.rpy [hunk 2]
```diff
@@ -1113,7 +1113,7 @@ init -1 python in jn_outfits:
     ))
     __register_wearable(JNHairstyle(
         reference_name="jn_hair_twintails_long",
-        display_name="Twintails long",
+        display_name="Long Hair with twintails",
```
Comment (Blizzardsev) [inline game/outfits.rpy:1116]: Almost! Sorry to be a pain, but do you think you could lowercase the "h" in "Hair"? Just so it's all consistent when we see it in a list with the others:

image

**Review by Blizzardsev — CHANGES_REQUESTED** [review]
Just a little thing

File: game/outfits.rpy (hunk 2 above)
Comment (RaionArt) [reply game/outfits.rpy:1116]: Damn you German genes! Sure, will not take long

**Comment by Blizzardsev** [comment]
Just adding here in case it gets lost: we should probably adjust the super messy hairstyle, so it works with the vanilla hairclips!

**Review by Blizzardsev — APPROVED** [review]
@Nemu-sus Any thoughts?

**Pull request merged**